src/components/Tabs/Tabs.stories.tsx

```diff
@@ -255,36 +169,45 @@ stories
         <TabPanel tab="Tab 1" tabKey="tab-1">
           <div>tab 1 content here</div>
         </TabPanel>
-        <TabPanel tab="Tab 2" tabKey="tab-2">
-          tab 2 content here
-        </TabPanel>
-        <TabPanel tab="Tab 3" tabKey="tab-3">
-          tab 3 content here
-        </TabPanel>
-        <TabPanel tab="Tab 32" tabKey="tab-35">
-          tab 3 content here
-        </TabPanel>
-        <TabPanel tab="Tab 42" tabKey="tab-45">
-          tab 4 content here
-        </TabPanel>
-        <TabPanel tab="Tab 52" tabKey="tab-55">
-          tab 5 content here
-        </TabPanel>
-        <TabPanel tab="Tab 12" tabKey="tab-16">
-          tab 1 content here
-        </TabPanel>
-        <TabPanel tab="Tab 22" tabKey="tab-26">
-          tab 2 content here
-        </TabPanel>
-        <TabPanel tab="Tab 32" tabKey="tab-36">
-          tab 3 content here
-        </TabPanel>
-        <TabPanel tab="Tab 42" tabKey="tab-46">
-          tab 4 content here
+        {TabPanels.slice(1, TabPanels.length)}
+      </Tabs>
+    </div>
+  ))
+  .add("Tabs with sort disabled", () => (
+    <div
+      style={{
+        border: "1px solid lightgray",
```

Comment:
Define common styles above and use them.

Reply:
done